include/color_spinor_field_order.h

```diff
@@ -264,11 +275,10 @@ namespace quda {
       const int stride;
       const int offset_cb;
       AccessorCB(const ColorSpinorField &field) :
-        stride(field.Stride()),
-        offset_cb((field.Bytes() >> 1) / sizeof(complex<Float>))
+        stride(field.VolumeCB()), offset_cb((field.Bytes() >> 1) / sizeof(complex<Float>))
```

Comment:
Should we change the variable name `stride` to `volume_cb`?

Reply:
We could do.  I left the `stride` variable there, since that's what it is at this point and the physical meaning isn't so important.  Aside: we'll likely address these variables again when we address large volume / 64-bit indexing.

Reply:
Fair enough---I think I saw `volume_cb` some other places, so this was also a comment on consistency, but I'm fine punting it until large volume / 64-bit indexing.